docs/source/changes.rst

```diff
@@ -3,18 +3,35 @@ History of changes
 
 .. currentmodule:: h11
 
-v0.8.1 (2018-04-14)
+v0.9.0 (2019-04-22)
 -------------------
 
 Bug fixes:
 
-* Always return headers as ``bytes`` objects (`#60
-  <https://github.com/python-hyper/h11/issues/60>`__)
 * Allow a broader range of characters in header values. This violates
   the RFC, but is apparently required for compatibility with
   real-world code, like Google Analytics cookies (`#57
   <https://github.com/python-hyper/h11/issues/57>`__, `#58
   <https://github.com/python-hyper/h11/issues/58>`__)
+* Constrain regex request-target matching (`#69
+  <https://github.com/python-hyper/h11/pull/69>`__)
+* Handle BrokenStreamError during shutdown grace period (`#71
+  <https://github.com/python-hyper/h11/pull/71>`__)
+
+Other changes:
+
+* Make all sentinel values inspectable by IDEs, and split
+  ``SEND_BODY_DONE`` into ``SEND_BODY``, and ``DONE`` (`#75
+  <https://github.com/python-hyper/h11/pull/75>`__)
```

Comment:
What's `SEND_BODY_DONE`?

Reply:
Before 57bf242a85c2b567a06ebaec0529689d8d3b8207 it was a state sentinel (`SEND_BODY_DONE = make_sentinel("SEND_BODY_DONE")`) - Thought this would be worth mentioning in case anyone used it directly.